NameGame/NameGame/Utilities/FilePath.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  FilePath.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/14/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import Foundation
+
+final class FilePath {
+    
+    static var tempDirectory: URL {
+        return URL(fileURLWithPath: NSTemporaryDirectory())
+    }
+    
+    static var imageDirectory: URL {
+        return tempDirectory.appendingPathComponent("images")
+    }
+    
+    static func path(for headshot: Headshot) -> URL {
+        return imageDirectory.appendingPathComponent("\(headshot.id).jpg")
+    }
+    
+    static func createDirectories() {
+        do {
+            try FileManager.default.createDirectory(
```

Comment:
Let's add a file manager check for the directory before we attempt to make it. Should clear up the logs when running the app multiple times.